### ISSUE TITLE
Library not loaded

### ISSUE THREAD

**Issue by ehunte4** [nanoporetech/dorado#317]
When I try to run Dorado after just downloading I get:
admin@ehlabehaxstudio ~ % /Applications/dorado-0.3.2-osx-arm64/bin/dorado 
dyld[3518]: Library not loaded: /opt/homebrew/opt/zstd/lib/libzstd.1.dylib
  Referenced from: <670420E3-434A-38DC-A606-BFE29A76BBEA> /Applications/dorado-0.3.2-osx-arm64/bin/dorado
  Reason: tried: '/opt/homebrew/opt/zstd/lib/libzstd.1.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/opt/homebrew/opt/zstd/lib/libzstd.1.dylib' (no such file), '/opt/homebrew/opt/zstd/lib/libzstd.1.dylib' (no such file), '/usr/local/lib/libzstd.1.dylib' (no such file), '/usr/lib/libzstd.1.dylib' (no such file, not in dyld cache)
zsh: abort      /Applications/dorado-0.3.2-osx-arm64/bin/dorado

In the Lib folder the libzstd.1.dylib file is an alias and points to libzstd.1.5.4.dylib

**Comment by tijyojwad** [comment]
Hi @ehunte4 - 

We're going to fix this in the next release by building all the dependencies into the binary itself. However, for now you can address the issue by providing the library path to your binary

```
export DYLD_LIBRARY_PATH=/Applications/dorado-0.3.2-osx-arm64/lib:$DYLD_LIBRARY_PATH
```

Then running `/Applications/dorado-0.3.2-osx-arm64/bin/dorado -h` should do the trick. Note that you'll need to either run the export command in every new terminal or you can store it in your `.bashrc` file.

Alternatively, to set that environment variable just for the command execution, you can also use
```
DYLD_LIBRARY_PATH=/Applications/dorado-0.3.2-osx-arm64/lib:$DYLD_LIBRARY_PATH /Applications/dorado-0.3.2-osx-arm64/bin/dorado -h
```

**Comment by tijyojwad** [comment]
Hi @ehunte4 - we just released v0.3.3. With that release, the binary includes these dependencies so you won't need to set any paths.
